### PR TITLE
perf: disable immutable & serializable checks for redux store in production

### PR DESCRIPTION
**Pull request by VictorVelarde**

### CODE REVIEW

File: template-sample-app/template/src/config/store.js [hunk 1]
```diff
@@ -71,20 +71,33 @@ const staticReducers = {
...
+      ignoredActions: ['carto/setViewportFeatures'],
+    },
+  };
+
+  const prodConfig = {
+    immutableCheck: false,
+    serializableCheck: false,
+  };
+
+  const isProductionEnv = process.env.NODE_ENV !== 'production';
```
Comment (AdriSolid) [inline template-sample-app/template/src/config/store.js:90]: Should be `process.env.NODE_ENV === 'production'` or am I missing something?

Comment (VictorVelarde) [reply template-sample-app/template/src/config/store.js:90]: 🤦🏻 .... No, the only missing thing is that I was tired... thx!

Comment (VictorVelarde) [reply template-sample-app/template/src/config/store.js]: Approve it if you want, please

**Review by AdriSolid — APPROVED** [review]
:rocket:

**Pull request merged**